torch/csrc/autograd/python_variable.cpp

```diff
@@ -291,7 +291,7 @@ int THPVariable_set_volatile(THPVariable *self, PyObject *obj)
 PyObject *THPVariable_get_output_nr(THPVariable *self)
 {
   auto& var = *self->cdata;
-  return PyInt_FromLong(var.output_nr);
+  return PyLong_FromLong(var.output_nr);
```

Comment:
I think it's better to leave it as `PyInt`. It's macroed to `PyLong` in Python 3, and is an `int` object in Python 2.